mobile/src/components/RoundButton/index.js

```diff
@@ -5,7 +5,7 @@ import { colors, fontFamilies, fonts } from '../../utils/theme'
 
 RoundButton.propTypes = {
     title: PropTypes.string.isRequired,
-    onPress: PropTypes.func.isRequired,
```

Comment:
Was there any place in the code that used this component without passing in `onPress`?